typings/balena-sdk.d.ts

```diff
@@ -571,26 +577,31 @@ declare namespace BalenaSdk {
 		release: NavigationResource<Release>;
 	}
 
+	// See: https://github.com/balena-io/resin-proxy/issues/51#issuecomment-274251469
+	interface OsUpdateActionResult {
+		// 'update_done' will be obsolete with https://github.com/balena-io/resin-proxy/pull/334
+		status: 'idle' | 'in_progress' | 'update_done' | 'resinhup_done';
```

Comment:
I think `resinhup_done` was changed to just `done` judging by a PR in the dashboard.

Reply:
Yea I also just saw that new change in the respective Proxy PR.
Will update this as well, thanks for the heads up :+1: